src/muxer/yamux/yamuxed_connection.cpp

```diff
@@ -9,6 +9,9 @@
 #include <libp2p/muxer/yamux/yamux_frame.hpp>
 #include <libp2p/muxer/yamux/yamux_stream.hpp>
 
+#define TRACE_ENABLED 0
+#include <libp2p/common/trace.hpp>
+
 using Buffer = libp2p::common::ByteArray;
```

Comment:
I understand that it's not your code, but Buffer should be reserved for common::Buffer.
I suggest to use `ByteArray = libp2p::common::ByteArray`